marlowe-playground-client/src/Marlowe/Parser.purs

```diff
@@ -245,7 +409,7 @@ atomValue =
 recValue :: Parser String Value
 recValue =
   (AvailableMoney <$> (string "AvailableMoney" **> accountId))
-    <|> (Constant <$> (string "Constant" **> parseTerm (maybeParens bigInteger)))
+    <|> (Constant <$> (string "Constant" **> bigIntegerTerm))
```

Comment:
Just checking - you meant to drop the parens on this, yes?

Reply:
yeah, it was a bug